packages/trip-form/src/components.story.tsx

```diff
@@ -11,23 +9,22 @@ import modeOptions from "./__mocks__/mode-options";
 import submodeOptions from "./__mocks__/submode-options";
 import trimet from "./__mocks__/trimet-styled";
 
-import englishMessages from "../i18n/en-US.yml";
 import { SettingsSelectorPanel } from "./styled";
 
 // Events
 const onChange = action("onChange");
 const onClick = action("onClick");
 const onQueryParamChange = action("onQueryParamChange");
 
-const intlDecorator = (Story: StoryType): ReactElement => (
-  <IntlProvider locale="en-US" messages={flatten(englishMessages)}>
+const decorator = (Story: StoryType): ReactElement => (
+  <div>
```

Comment:
Is a surrounding div necessary? Would <> work?

Reply:
That's possible indeed (see https://github.com/opentripplanner/otp-ui/pull/355/commits/2108869a6cf786fc78255b1bd12746ccf7523128).